internal/kubehelper/client.go

```diff
@@ -11,8 +12,19 @@ import (
 
 func createKubeClient(timeout int64) (*kubernetes.Clientset, error) {
 	var cliKubeconfig string
-	flag.StringVar(&cliKubeconfig, "kubeconfig", "", "absolute path to the kubeconfig file")
-	flag.Parse()
+	for i := 1; i < len(os.Args)-1; i++ {
+		if os.Args[i] == "--kubeconfig" {
+			cliKubeconfig = os.Args[i+1]
+			break
```

Comment:
Sorry, I just realized… the user can totally provide this multiple times.
```
$ kubectl --kubeconfig /tmp/foo --kubeconfig ~/.kube/config config current-context
rancher-desktop
```
Yes, that's dumb… but quite possible if this was a non-interactive script.  It appears to take the _last_ `--kubeconfig` value.

Reply:
Thanks, fixed by removing the `break` statements